### PR TITLE
Use new infra for sierra_adapter

### PR DESCRIPTION
**Pull request by kenoir**
### What is this PR trying to achieve?

Update the sierra_adapter to use the new infra modules.

### Who is this change for?

Folk who want simpler infra!

### CODE REVIEW

File: sierra_adapter/terraform/pipeline_items.tf [hunk 1]
```diff
@@ -50,42 +49,35 @@ module "items_to_dynamo" {
...
-  release_id = "${var.release_ids["sierra_item_merger"]}"
+  release_id    = "${var.release_ids["sierra_item_merger"]}"
 
   merged_dynamo_table_name = "${local.vhs_table_name}"
 
-  updates_topic_name = "${module.items_to_dynamo.topic_name}"
-
-  cluster_name = "${module.sierra_adapter_cluster.cluster_name}"
-  vpc_id       = "${module.vpc_sierra_adapter.vpc_id}"
+  updates_topic_name = "${module.bibs_reader.topic_name}"
```
Comment (alexwlchan) [inline sierra_adapter/terraform/pipeline_items.tf]: This topic is wrong.

**Comment by alexwlchan** [comment]
I’m going to review this now.

File: sierra_adapter/terraform/merger/iam_role_policy.tf [hunk 2]
```diff
@@ -3,6 +3,11 @@ resource "aws_iam_role_policy" "vhs_full_access_policy" {
   policy = "${var.vhs_full_access_policy}"
 }
 
+resource "aws_iam_role_policy" "allow_read_from_windows_q" {
```
Comment (alexwlchan) [inline sierra_adapter/terraform/merger/iam_role_policy.tf]: Either this name is wrong, or the queue is wrong.

File: sierra_adapter/terraform/merger/main.tf [hunk 3]
```diff
@@ -0,0 +1,41 @@
+data "aws_ecs_cluster" "cluster" {
+  cluster_name = "${var.cluster_name}"
+}
+
+module "sierra_merger_service" {
+  source = "git::https://github.com/wellcometrust/terraform.git//ecs/modules/service/prebuilt/sqs_scaling?ref=v11.0.0"
+
+  service_name       = "sierra_${local.resource_type_singular}_merger_v2"
```
Comment (alexwlchan) [inline sierra_adapter/terraform/merger/main.tf]: Why `_v2`?

File: sierra_adapter/terraform/merger/iam_role_policy.tf (hunk 2 above)
Comment (kenoir) [reply sierra_adapter/terraform/merger/iam_role_policy.tf]: name is wrong

File: sierra_adapter/terraform/merger/main.tf (hunk 3 above)
Comment (kenoir) [reply sierra_adapter/terraform/merger/main.tf]: because i forget to delete it

**Pull request merged**